CRM/Financial/Form/Payment.php

```diff
@@ -37,6 +37,7 @@ class CRM_Financial_Form_Payment extends CRM_Core_Form {
    */
   protected $_paymentProcessorID;
   protected $currency;
```

Comment:
we don't need the changes in this form if we are not trying to overload the payment form with the edit transaction form